course/templates/course/course_cards.html

```diff
@@ -0,0 +1,50 @@
+{% load i18n %}
+{% load course %}
+
+{% for instance in instances %}
+{% with instance_url=instance|url %}
+	<div class="col-sm-6 col-md-4 col-lg-3">
+		<div class="frontpage-course card">
+            <div class="card-clickable">
+                <a href="{{ instance_url }}">
+                    <div class="course-image" aria-hidden="true">
+                        {% if instance.image %}
+                            <img class="card-img-top" src="{{ instance.image.url }}" alt="" />
+                        {% else %}
+                            <div class="card-img-top">
+                                <i class="glyphicon glyphicon-book"></i>
+                            </div>
+                        {% endif %}
+                    </div>
+                    <div class="card-body">
+                        <h3 class="card-title" lang="{{ instance.language }}">
+                            {{ instance.course.name|parse_localization }}<br />
+                        </h3>
+                        <p class="card-subtitle" lang="{{ instance.language }}">
+                            {{ instance.instance_name|parse_localization }}
+                        </p>
+                        <p class="card-text">
+                            {{ instance.course.code }}
+                            {% if not instance.visible_to_students %}
+                                <span class="label label-danger">{% trans "Hidden from students" %}</span>
+                            {% endif %}
+                            <br />
```

Comment:
As you are moving these to a new file, you can change these to `<br>` (i.e. drop `/` from it). Does not matter though.

Reply:
Alright, good to know for future :)